rca/scholarships/birdbath.py

```diff
@@ -0,0 +1,32 @@
+from faker import Faker
+
+from rca.scholarships.models import ScholarshipEnquiryFormSubmission
+from rca.utils.birthbath import BaseModelAnonymiser
+
+fake = Faker()
+
+
+class ScholarshipEnquiryFormSubmissionAnonymiser(BaseModelAnonymiser):
+    """
+    Anonymise ScholarshipEnquiryFormSubmission
+    """
+
+    model = ScholarshipEnquiryFormSubmission
+
+    anonymise_fields = [
+        "first_name",
+        "last_name",
+        "email",
+        "rca_id_number",
+    ]
+
+    def get_anonymous_field_value(self, field_name, obj):
+        if field_name == "first_name":
+            return fake.first_name()
+        if field_name == "last_name":
+            return fake.last_name()
+        if field_name == "email":
+            return fake.ascii_email()
```

Comment:
As above comment. Perhaps fake email can have `.fake` appended to it.

Reply:
And delete them here too as above?

Reply:
Yes, sounds good.